lib/cmd/fh3/app/_baseCloudRequest.js

```diff
@@ -5,6 +6,7 @@ var util = require('util');
 var fhc = require("../../../fhc");
 var request = require('request').defaults({'proxy': fhc.config.get("proxy")});
 var url = require('url');
+var util = require('util');
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Rename "util" as this name is already used in declaration at line 5. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://metrics.skunkhenry.com/coding_rules#rule_key=javascript%3AS2814)